pkg/interop/native/neo/neo.go

```diff
@@ -1,5 +1,5 @@
 /*
-Package neo provides interface to NeoToken native contract.
+Package neo provides an interface to a NEOToken native contract.
```

Comment:
It's actually `NeoToken`, it's the name of the token contract.

Reply:
Thanks